src/extras/geometries/CylinderBufferGeometry.js

```diff
@@ -20,6 +20,7 @@ THREE.CylinderBufferGeometry = function( radiusTop, radiusBottom, height, radial
 	};
 
 	var scope = this;
+	var twoPi = 2.0 * Math.PI;
```

Comment:
Indeed! I don't think the variable is necessary here.

@abelnation Is this something you can handle? :wink: